components/serverless/Makefile

```diff
@@ -109,7 +109,7 @@ else
 	@echo "Image signing skipped"
 endif
 
-install-manager-k3d: build-image-function-controller
+install-manager-k3d: build-image-function-controller disable-operator
```

Comment:
why?

Reply:
Current installation process broke this target. This target is intended to use for local development. The fix is simple and I would keep it.